op-chain-ops/genesis/config.go

```diff
@@ -705,13 +709,21 @@ func NewL2StorageConfig(config *DeployConfig, block *types.Block) (state.Storage
 		"_initializing": false,
 		"messenger":     predeploys.L2CrossDomainMessengerAddr,
 	}
+	batcherHash := common.Hash{}
+	if config.InitialBatcherVersion == derive.BatchV1Type {
+		batcherHash = config.BatchSenderAddress.Hash()
+	} else if config.InitialBatcherVersion != derive.BatchV2Type {
```

Comment:
Why don't we handle the case` if config.InitialBatcherVersion == derive.BatchV2Type `?

Reply:
We effectively handle it with line 718... I could put a no-op in there, since we *should* always do that, no matter what version we're on.